en/guide/missions.md

```diff
@@ -1,6 +1,6 @@
 # Missions
 
-The DroneCore Mission API allows you to create, upload, download, run, pause, restart, jump to item in, and track missions. Missions can have multiple "mission items", each which may specify a position, altitude, fly-through behaviour, camera action, gimbal position, and the speed to use when traveling to the next position.
+The DroneCore Mission API allows you to create, upload, download, import from *QGroundControl*, run, pause, restart, jump to item in, and track missions. Missions can have multiple "mission items", each which may specify a position, altitude, fly-through behaviour, camera action, gimbal position, and the speed to use when traveling to the next position.

```

Comment:
>Missions can have multiple "mission items"

Shouldn't this be **A mission can have multiple "mission items"** ?

Reply:
No, it is correct (what you have suggested is also fine).